projects/Rockchip/devices/RK3288/linux/default/linux.arm.conf

```diff
@@ -5780,7 +5780,7 @@ CONFIG_CRYPTO_LIB_SHA256=y
 CONFIG_CRYPTO_HW=y
 # CONFIG_CRYPTO_DEV_ATMEL_ECC is not set
 # CONFIG_CRYPTO_DEV_ATMEL_SHA204A is not set
-CONFIG_CRYPTO_DEV_ROCKCHIP=m
+# CONFIG_CRYPTO_DEV_ROCKCHIP is not set
```

Comment:
why would you drop crypto driver?

Reply:
see commit msg: "Using this driver crashes the SoC: drop it."

Reply:
sorry, I missed it.